pulse_actions/worker.py

```diff
@@ -53,7 +54,10 @@ def handler_with_dry_run(data, message):
     LOG.info('Listening on %s, with topic %s', exchange, topic)
 
     while True:
-        pulse.listen()
+        try:
```

Comment:
We might need a better way of dealing with exceptions, but this was very important to not have to restart Heroku every time something went wrong and collect different problems.